modules/coreboot

```diff
@@ -93,7 +93,7 @@ $(eval $(call coreboot_module,purism,24.02.01))
 
 # MSI and Nitropad NV41 / NS50 boards are based on Dasharo coreboot port
 coreboot-dasharo_repo := https://github.com/dasharo/coreboot
-coreboot-dasharo_commit_hash := 3a9aa3a4692f3dd49732f5b4e3ec54be385f0969
+coreboot-dasharo_commit_hash := 38215f5a2bb3ea8ead10bf9c481caeb07d3a19ad
```

Comment:
@macpijan : good commit for nv41/ns50/msi boards?

Reply:
They are close enough, should be relatively safe to use it. Definitely MSI requires the newer revision, so let's update it.